proto/redpanda/api/console/v1alpha1/debug_bundle.proto

```diff
@@ -166,6 +166,10 @@ message GetDebugBundleStatusResponse {
   }
 
   repeated DebugBundleBrokerStatus broker_statuses = 1;
+
+  // true IFF all broker statuses return
+  // "debug bundle process" not started error code
+  bool debug_bundle_process_not_started = 2;
```

Comment:
NIT: I'm not a friend of negated bools. I'd prefer to call if `debug_bundle_process_started` instead. WDYT?

Reply:
Changed to `has_debug_process`.